test/todo_or_die_test.rb

```diff
@@ -74,6 +74,16 @@ def test_when_rails_is_a_thing_and_is_production
     MSG
   end
 
+  def test_todo_or_die_file_path_removed_from_backtrace
+    Timecop.travel(Date.civil(2200, 2, 4))
+
+    error = assert_raises(TodoOrDie::OverdueTodo) {
+      TodoOrDie("Fix stuff", by: Date.civil(2200, 2, 4))
+    }
+
+    assert_empty(error.backtrace.select {|line| line.match?(TodoOrDie::FILE_PATH_REGEX) })
```

Comment:
I think I'd rather the test explicitly match on `/todo_or_die/` than reuse the production-scope regex, since (if that's broken) this test will keep passing.